src/features/cellnav/js/cellnav.js

```diff
@@ -954,6 +954,16 @@
                     }
                   }
                 });
+
+                focuser.on('focusout', function(evt) {
+                  uiGridCtrl.cellNav.clearFocus();
+                });
+
+                $scope.$on(uiGridCellNavConstants.CELL_NAV_EVENT, function(evt, rowCol) {
```

Comment:
This is silly confusing. Basically, due to a chain of events a mouse click gets swallowed and a focus event does not occur again when refocusing a column. Due to this we have to listen to cell nav from any source and make sure that the focuser is focused.